renovate.json

```diff
@@ -1,6 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["local>Altinn/renovate-config"],
+  "separateMinorPatch": true,
+  "separateMultipleMajor": true,
```

Comment:
Gives separate PRs for major, minor and patch bumps. I've put this globally here, but could also put it inside a package rule for `src/App/**`